komga-webui/src/locales/en.json

```diff
@@ -500,9 +500,9 @@
       "UNSUPPORTED": "Unsupported"
     },
     "reading_direction": {
-      "LEFT_TO_RIGHT": "Left to right",
```

Comment:
Better for quick cognition

Reply:
Since you can't have one for webtoon, we lack consistency. The icon usage should be managed by the UI, not the translations.